### DISCUSSION TITLE
Is this being sunsetted?

### DISCUSSION THREAD

**Discussion by joezimjs** [vuejs/devtools-v6#2075] [Q&A]
There have only been two small commits since the end of January. There are well over 400 Issues sitting and waiting to fixed and 40 pull requests that have been waiting a minimum of 3 weeks to be merged. Along with this, a [new dev tool](https://github.com/webfansplz/vite-plugin-vue-devtools) is under development, so it can be tempting to let this sit by the wayside. 

I know the vast majority of development has been done by @Akryum, and I would be very understanding of him maybe being burnt out or busy with other projects, but we can't let this project just die out. There are too many people like me who have Vue projects that cannot or will not be changed to use Vite, so the new devtool isn't useful and there are too many existing issues with this devtool to just let it sit (I can't inspect the refs/computeds inside my component and I have to hit refresh to see the updates, which means this tool has lost a lot of its primary usefulness).

So, what does the near future of this project look like? Is anyone aware that this appears to be dying?

**Comment by whitespacecode** [comment]
Thanks for the new dev tool. I hope this would be developed further!
